packages/p2p/src/components/my-profile/my-profile-header/my-profile-header.tsx

```diff
@@ -28,7 +28,8 @@ const MyProfileHeader = () => {
         },
     ];
 
-    const onChangeTab = event => my_profile_store.setActiveTab(event.target.value);
+    const onChangeTab = (event: { target: { value: string; name: string } }) =>
```

Comment:
```suggestion
    const onChangeTab = (event: React.ChangeEvent<HTMLInputElement>) =>
```

Reply:
maybe the type in the button-toggle component is wrong :thinking:

Reply:
Yeah... i initially tried that typing but typescript was complaining, let me check the button-toggle and see if its possible to change its type

Reply:
ok so many places use this component, i might break all the components using it 😆

Reply:
Let's leave it like this :smile: